train_hierarchialXC.py

```diff
@@ -1,4 +1,4 @@
-from sklearn import svm
+from sklearn import multiclass, svm
```

Comment:
svm is unused. Pop it off.

Reply:
It is used. Line 78.

Reply:
Not in this file, right?

Reply:
Oh sorry, you are right. But first please see the message below. It should have been used, but has not.